lib/pbench/server/auth/auth.py

```diff
@@ -168,13 +250,13 @@ def verify_third_party_token(auth_token: str, oidc_client: OpenIDClient) -> bool
 
         if not oidc_client.TOKENINFO_ENDPOINT:
```

Comment:
At line 215 we are testing to see if `oidc_client` is a "True" value.  Does that mean looking for the `TOKENINFO_ENDPOINT` attribute here will raise an `AttributeError` when `oidc_client` is a "False" value?

Reply:
Yeah, either I should remove that check to see if the `oidc_client` is `None` on line 215 or keep it in both places. However, there is no point in doing the check for `None` because if `oidc_client` is None we would have long failed before this step (at the server startup).